x-pack/plugins/ml/common/util/anomaly_utils.ts

```diff
@@ -109,6 +109,13 @@ function getSeverityTypes() {
   });
 }
 
+/**
+ * Return formatted severity score.
+ */
+export function getFormattedSeverityScore(score: number) {
+  return score < 1 ? '< 1' : score.toFixed(2);
```

Comment:
The anomaly score needs to be floored for scores of 1 and greater. We don't want the decimal places.

Reply:
Fixed in https://github.com/elastic/kibana/pull/79315/commits/6e6321582c7ce3c830749aa66a81e1dc09a24355